content/en/docs/installation/cli.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Install the notation CLI
+description: Install the notation CLI on Linux, macOS, and Windows
+weight: 1
+---
+
+## Download the `notation` CLI 
+
+Download the latest CLI for your platform from the [Releases page](https://github.com/notaryproject/notation/releases).
+
```

Comment:
Can we provide a specified release and commands for users to easily copy & paste into the installation guide? It's similar to this [guide](https://github.com/notaryproject/notation#notation-quick-start).

Reply:
@FeynmanZhou this will differ depending on your platform and will constantly change for every release. This generic instruction gives users enough details for downloading the binary and the subsequent sections are platform-specific steps.

Reply:
Agree with @zr-msft - less maintenance of doc.  That said - i do think we should: 
1. cleanup the release page of the developer builds.  Filed this issue: https://github.com/notaryproject/notation/issues/369
2. add notation to common install tools (i.e. brew, chocolately, etc)